.github/workflows/trivy.yaml

```diff
@@ -43,10 +43,3 @@ jobs:
         run: |
           cd hack
           python3 trivy_scan.py
-
```

Comment:
Just leave it commented out, but do not delete


      # Upload the artifact
      - name: Upload trivy scanned_results
        uses: actions/upload-artifact@v4
        with:
          name: trivy_scanned_results
          path: image_lists/